src/FlyWithLua.cpp

```diff
@@ -6681,15 +6680,15 @@ bool ReadAllScriptFiles()
                 oss_ScrPathAndName << scriptDir << "/" << FileToLoad;
                 auto ScrPathAndName = oss_ScrPathAndName.str();
 
-                logMsg(logToDevCon, std::string("FlyWithLua Info: Start loading script file ").append(ScrPathAndName.c_str()));
+                logMsg(logToDevCon, std::string("FlyWithLua Info: Start loading script file " + ScrPathAndName));
```

Comment:
`std::string` is redundant here. Same below.

The expression `"FlyWithLua Info: Start loading script file " + ScrPathAndName` already has std::string type because `+` between `std::string` and `char*` [is overloaded](http://www.cplusplus.com/reference/string/string/operator+/) and returns `std::string`.

Also worth mentioning that C++ supports [implicit type conversions](https://www.learncpp.com/cpp-tutorial/44-implicit-type-conversion-coercion/). Specifically, if a `char*` string is passed as a function argument which expects `std::string`, the compiler will automatically generate code to convert `char*` to `std::string` for you.